[BUG][node-express-server] .eslintrc.json extends "airbnb" but should be "airbnb-base"

<!--
Please follow the issue template below for bug reports and feature requests.
Also please indicate in the issue title which language/library is concerned. Eg:  [JAVA] Bug generating foo with bar 
-->

##### Description

<!-- describe what is the question, suggestion or issue and why this is a problem for you. -->
The `.eslintrc.json` generated from `eslintrc.mustache` attempts to extend the `airbnb` config plugin. This is incompatible with the `eslint-config-airbnb-base` included as a dev dependency in the `package.json` generated by `package.mustache`.  ESLint generates an error: `Cannot find module 'eslint-config-airbnb'`.

##### openapi-generator version

<!-- which version of openapi-generator are you using, is it a regression? -->
Discovered on `stable 4.3.1`, confirmed on `master`. Looks like it was introduced in `4.3.0` by https://github.com/OpenAPITools/openapi-generator/pull/5675.

##### OpenAPI declaration file content or url

Reproduced with the Petstore demo example at [https://raw.githubusercontent.com/openapitools/openapi-generator/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml](https://raw.githubusercontent.com/openapitools/openapi-generator/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml)

##### Command line used for generation

```
java -jar ~/dev/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar generate \
    --generator-name nodejs-express-server \
    --input-spec https://raw.githubusercontent.com/openapitools/openapi-generator/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml \
    --output build/
```

##### Steps to reproduce

<!-- unambiguous set of steps to reproduce the bug.-->
1. Clone the repo, and build with `mvn clean install`
2. Run the `generate` command above
3. Run ESLint on the generated `expressServer.js`: `npx eslint expressServer.js` or `yarn run eslint expressServer.js`

__Note:__ Once this issue is fixed, ESLint finds some errors that I consider out of scope of this issue — but after this is fixed, it is at least able to run.

##### Related issues/PRs

I searched but didn't  find any. 

##### Suggest a fix/enhancement

**This is a simple fix:** Updating `eslintrc.mustache` to extend `airbnb-base` resolves this.

It makes sense to stick with `eslint-config-airbnb-base`, which "provides Airbnb's base JS .eslintrc (without React plugins)" (quoting eslint-config-airbnb-base's [description](https://www.npmjs.com/package/eslint-config-airbnb-base)).

I'm working on a PR.

--- a/modules/openapi-generator/src/main/resources/nodejs-express-server/eslintrc.mustache
+++ b/modules/openapi-generator/src/main/resources/nodejs-express-server/eslintrc.mustache
@@ -1,7 +1,7 @@
 // Use this file as a starting point for your project's .eslintrc.
 // Copy this file, and add rule overrides as needed.
 {
-  "extends": "airbnb",
+  "extends": "airbnb-base",
   "rules": {
     "no-console": "off"
   }

